File: 2113-FindTheKthLargestIntegerInTheArray/2113-FindTheKthLargestIntegerInTheArray.py
# Last updated: 12/28/2025, 4:52:38 PM
class Solution:
    def kthLargestNumber(self, nums: List[str], k: int) -> str:
        l=[]
        for i in nums:
            i=int(i)
            l.append(i)
        for i in range(k-1):
            l.remove(max(l))
        return str(max(l))

File: 2113-FindTheKthLargestIntegerInTheArray/2113-FindTheKthLargestIntegerInTheArray.java
// Last updated: 12/28/2025, 4:52:40 PM
import java.math.BigInteger;
class Solution {
    public String kthLargestNumber(String[] nums, int k) {
      PriorityQueue<BigInteger> minHeap = new PriorityQueue<>();
      for(String num:nums){
          BigInteger negateNum = new BigInteger(num).negate();
          minHeap.offer(negateNum);
      }  
      while(k>1){
          minHeap.poll();
          k--;
      }
      return String.valueOf(minHeap.peek().negate());
    }
}